guides/common/modules/proc_reset-custom-ssl-certificate-to-default-self-signed-certificate-on-smartproxy.adoc

```diff
@@ -0,0 +1,43 @@
+[id="reset-custom-ssl-certificate-to-default-self-signed-certificate-on-{smart-proxy-context}_{context}"]
+= Reset {customssl} certificate to default self-signed certificate on {SmartProxyServer}
+
+Use this procedure to reset {customssl} certificate to default self-signed certificate on your {SmartProxyServer}.
+
+.On {ProjectServer}
+. Generate a new certificate tar bundle for {SmartProxyServer}.
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {certs-generate} \
+--foreman-proxy-fqdn _{smart-proxy-context}.example.com_ \
+--certs-tar /root/_{smart-proxy-context}.example.com_-certs.tar.gz \
+--certs-update-all
+----
+. Note down the {foreman-installer} command generated by the above command.
+. Copy the new certificate tar bundle to {SmartProxyServer}.
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# scp /root/_{smart-proxy-context}.example.com_-certs.tar.gz \
+root@_{smart-proxy-context}.example.com_:/root/_{smart-proxy-context}.example.com_-certs.tar.gz
+----
+
+.On {SmartProxyServer}
+. Ensure you received the certificate tar file from {ProjectServer}.
+. Update the new `katello-ca-consumer-latest.noarch.rpm` package:
```

Comment:
An update of the katello-ca-consumer is needed [ whether on capsule or any managed hosts ] to populate the new SSL certificate information on the hosts. 

For managing the hosts guide, I see that we have skipped the usage of `katello-ca-consumer update` but instead suggested to re-register them using the Global Registration method.

If so, Then for capsule\external smart-proxy also the same should be applicable instead of using katello-ca-consumer as Capsule\External Smart-proxy is essentially a Client of Satellite\Foreman only.

Reply:
So can we go with the `Global Registration` way in the SmartProxy as well? Instead of `katello-ca-consumer`, because it is deprecated.

Reply:
That is the only way forward if the goal is to get away from katello-ca-consumer but still, we need to reflect the changes related to SSL certs. 

Once both https://bugzilla.redhat.com/show_bug.cgi?id=2185634 and https://bugzilla.redhat.com/show_bug.cgi?id=2124052 are fixed, then this part can further be improved i believe.

Reply:
As per our talk, we are keeping `katello-ca-consumer` for SmartProxy as of now until the above BZs are closed.